js/portofolio.js

```diff
@@ -12,6 +109,91 @@ function close() {
   menuItem.classList.remove('visible_nav_menu');
   menuItem.classList.toggle('hidden');
 }
+
+function closepopup() {
+  modal.classList.remove('visible_modal');
+  modal.classList.toggle('hidden_modal');
+}
+
+function showpopup(index) {
+  let modalcontent = `${'<div class="modal_popup">'
+            + '<div class="modal_container">'
+              + '<div class="modal_close_button" onclick="closepopup()"><i class="fa-solid fa-xmark "></i></div>'
+            + '<div class="modal_image"  onclick="closepopup()"> </div>'
+            + '<div class="desktop_title">'
+              + '<h2 class="modal_title">'}${dataSource[index].card_popup_title}</h2>`
+            + '<div class="buttons for_desktop">'
+              + `<a href="${dataSource[index].card_live_link}"><button class="left_button modal_button">See live <img src="assets/see_live_icon.svg" alt="see live icon"> </button></a>`
+              + `<a href="${dataSource[index].card_source_link}"><button class="left_button modal_button">See Source <img src="assets/github_icon.svg" alt="github icon"> </button></a>`
+            + '</div></div><ul class="modal_popup_technologies">';
+  dataSource[index].card_technologies.forEach((element) => {
+    modalcontent += `<li class="work_language_item">${element}</li>`;
+  });
+  modalcontent += `</ul><p class="popup_description">${dataSource[index].card_description}</p><div class="buttons for_mobile">`
+                + `<a href="${dataSource[index].card_live_link}"><button class="left_button modal_button">See live <img src="assets/see_live_icon.svg" alt="see live icon"> </button></a>`
+                + `<a href="${dataSource[index].card_source_link}"><button class="left_button modal_button">See Source <img src="assets/github_icon.svg" alt="github icon"> </button></a>`
```

Comment:
- [ ] Please remove the black border around the model buttons and add some border radius to match the Figma design.

### Your design
![image](https://user-images.githubusercontent.com/83803414/189311189-0b3cc0b6-c4a0-46c7-aa9e-1734bda4f03b.png)

### Figma design
![image](https://user-images.githubusercontent.com/83803414/189311242-7560a6ed-f6c7-4903-9b6e-3153f652925a.png)